source/_integrations/zwave_js.markdown

```diff
@@ -707,7 +705,7 @@ You can also keep track of the roadmap for the Z-Wave integration [here](https:/
 
 #### Which Z-Wave controller should I buy?
 
-Z-Wave supports all known 500 and 700 series Z-Wave controllers. If you are just starting out, we recommend that you purchase a 500 series controller.
+Z-Wave supports all known 500 and 700 series Z-Wave controllers. If you are just starting out, we recommend that you purchase a 700 series controller.
```

Comment:
@AlCalzone We're ok with changing the recommendation back to the 700 stick now, right?

Reply:
Yeah we should mention though that the firmware must be upgraded to 7.17.2 or higher.